### ISSUE TITLE
Skymaps.py

### ISSUE THREAD

**Issue by Areustle** [fermi-lat/ScienceTools#18]
Need to include missed `skymaps.py` file in fermitools. This is generated by swig in the skymaps submodule. It depends on the numpy C api, so numpy must be made an explicit build-time dependency in fermi-lat/Fermitools-conda.

**Comment by Areustle** [comment]
A new build which attempts to address this issue is now on anaconda cloud. It is fermitools 2.2.9.

This build includes Numpy as an explicit build-time requirement. That is a modest issue for cross-compiled builds, which is addressed using CMAKE's FindPackage(Python3) numpy support module. At build time CMAKE should be invoked with an additional flag `-DPython3_NumPy_INCLUDE_DIR=...` pointing to the target numpy include directories on cross-compiled platforms. This update has been added to the Fermitools-conda STIssue18 branch.

**Comment by Areustle** [comment]
@donhorner reports segfault when importing skymaps in fermitools 2.2.9. Does not occur on linux.

**Comment by Areustle** [comment]
The import segfault on macs was due to the lack of flags passed to the linker during linktime: `-bundle` and `-flat_namespace`. I added this to the CMake file for Apple builds and that resolved the segfault issue.

Now the numpy API is a build-time dependency. On cross-compiled builds the `-DPython3_NumPy_INCLUDE_DIR=` flag must be specified. I also added a numpy version range to the list of runtime dependencies, specifically 1.18 - 2.0

The latest dev version (2.2.11) fixes these issues.
